### PR TITLE
add class to read google spreadsheets, add comments.

### PR DESCRIPTION
**Pull request by thiagopf**

### CODE REVIEW

File: pyheaderfile/__init__.py [hunk 1]
```diff
@@ -527,6 +679,118 @@ def _import(self):
...
+        :return: {header: value}
+        """
+        if isinstance(self.header[y], tuple):
+            header = self.header[y][0]
+        else:
+            header = self.header[y]
+        # need to add 1 to the index for the google spreadsheet reads from the index 1
+        # TODO(dmvieira, thiagopf) improve it
+        x += 1
+        y += 1
```
Comment (dmvieira) [inline pyheaderfile/__init__.py]: It sounds good! You're doing right

File: pyheaderfile/__init__.py [hunk 2]
```diff
@@ -527,6 +679,118 @@ def _import(self):
...
+        self._sheet.update_cell(x, y, value)
+
+    def save(self, path=None):
+        """
+
+        :param path:
+        :return:
+        """
+        raise NotImplementedError
```
Comment (dmvieira) [inline pyheaderfile/__init__.py]: Don't you save?

File: pyheaderfile/__init__.py [hunk 3]
```diff
@@ -527,6 +679,118 @@ def _import(self):
...
+        Writing value in the cell of x+1 and y+1 position
+        :param x: line index
+        :param y: coll index
+        :param value: value to be written
+        :return:
+        """
+        # need to add 1 to the index for the google spreadsheet reads from the index 1
+        # TODO(dmvieira, thiagopf) improve it
+        x += 1
+        y += 1
```
Comment (dmvieira) [inline pyheaderfile/__init__.py]: you can call a function to do it, but I still think it is good

File: pyheaderfile/__init__.py [hunk 4]
```diff
@@ -527,6 +679,118 @@ def _import(self):
...
+        """
+        Read the sheet, get value the header, get number columns and rows
+        :return:
+        """
+        if self.sheet_name and not self.header:
+            self._sheet = self._file.worksheet(self.sheet_name.title)
+            self.ncols = self._sheet.col_count
+            self.nrows = self._sheet.row_count
+            for i in range(0, self.ncols):
+                self.header = self.header + [self._sheet.cell(1, i+1).value]
```
Comment (dmvieira) [inline pyheaderfile/__init__.py]: here you can range from 1 to self.ncols... dont need i+1

**Comment by dmvieira** [comment]
Great work!

File: pyheaderfile/__init__.py (hunk 4 above)
Comment (thiagopf) [inline pyheaderfile/__init__.py]: need but if do not have to add 1 to self.ncols

File: pyheaderfile/__init__.py (hunk 2 above)
Comment (thiagopf) [inline pyheaderfile/__init__.py]: not need to save

**Pull request merged**